src/components/pages/NewFeedPage/index.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+
+import { useHistory } from "react-router-dom";
+
+import { NewFeedModal } from "../../templates";
+import { NewFeed } from "../../organisms";
+
+function NewFeedPage() {
+  const history = useHistory();
+  const [isOpen, setIsOpen] = useState(true);
+
+  const handleModalClose = () => {
+    setIsOpen(false);
+    history.push("/");
+  };
+
+  if (!isOpen) return null;
```

Comment:
다른 코드 스타일과 일관성을 맞춰주는 것이 좋을 것 같습니다🙇‍♀️
여기서 해당 분기문이 추가된 이유가 있을까요? 함수 안에 감싸져있지 않아서 호출해서 값을 받는 것도 아닌 것 같고 해당 로직은 실행만되고 다른 코드에 아무런 영향을 주지 않을 것 같습니다...!

```suggestion
  if (!isOpen) {
    return null;
   }
```

Reply:
사용하지 않는 페이지라서 삭제하겠습니다